data/openai

```diff
@@ -2,6 +2,11 @@
 ai.com
 openai.com
 
-# CDN
+# CDN & API
 full:chat.openai.com.cdn.cloudflare.net
 full:openaiapi-site.azureedge.net
+full:production-openaicom-storage.azureedge.net
+full:openaicom-api-bdcpf8c6d2e9atf6.z01.azurefd.net
+full:openaicomproductionae4b.blob.core.windows.net
+full:openaicom.imgix.net
+full:o33249.ingest.sentry.io
```

Comment:
any source?
// BTW, pls sort.

Reply:
Collected from my log.
sort by what? eTLD? I'm trying to sort by service provider, e.g. CF, MS, other small company

Reply:
sort by ASCII code.

Reply:
> Collected from my log.

In that case, please provide the log for further reviewing.